app/assets/javascripts/manage_submissions.js

```diff
@@ -109,7 +115,10 @@ $(document).ready(function () {
               </td>
               ${submission.problems.
               map((problem) =>
-                `<td class="submissions-td">${data.scores[submission.id][problem.id]?.['score']}</td>`
+                data.scores[submission.id] ?
```

Comment:
You might want to strengthen the check to `data.scores[submission.id]?.[problem.id]`. Otherwise, submissions with no score for a particular problem would still show undefined (assuming you make the dash change below).

Reply:
Changed.